src/services/ExposureNotificationService/ExposureNotificationServiceProvider.tsx

```diff
@@ -50,9 +48,8 @@ export const ExposureNotificationServiceProvider = ({
         storage || AsyncStorage,
         secureStorage || RNSecureKeyStore,
         exposureNotification || ExposureNotification,
-        metrics.service,
       ),
-    [backendInterface, exposureNotification, i18n, metrics.service, secureStorage, storage],
```

Comment:
This was the line causing the bug - `metrics.service` must be changing whenever bluetooth is toggled on/off, causing a new instance of `ExposureNotificationService` to be created.